tilde/.hammerspoon/app-config/mailmate.lua

```diff
@@ -9,7 +9,7 @@ local function mailmateCall(name, eventType, app)
         end
     end
 
-    if name == "MialMate" and eventType == hs.application.watcher.terminated then
+    if name == "MailMate" and eventType == hs.application.watcher.terminated then
         _log("Stretchly closed")
```

Comment:
```suggestion
        _log("MailMate closed")
```
i think this should also here MailMate